eng/common/dotnet-install.sh

```diff
@@ -70,7 +70,7 @@ case $cpuname in
     ;;
 esac
 
-dotnetRoot="$repo_root/.dotnet"
+dotnetRoot="${repo_root}.dotnet"
```

Comment:
>repo_root [](http://example.com/codeflow?start=14&length=9)

does the repo_root always appended with the path separator?

Reply:
Feedback for https://github.com/dotnet/arcade/pull/7498